### PR TITLE
gdebi: add page

### PR DESCRIPTION
**Pull request by prophet1996**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/gdebi.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`gdebi {{package.deb}}`
+
+- Display the program version:
+
+`gdebi --version`
+
+- Do not show progress information:
+
+`gdebi {{package.deb}} -q`
```
Comment (bl-ue) [inline pages/linux/gdebi.md:16]:
```suggestion
`gdebi {{path/to/package.deb}} --quiet`
```

File: pages/linux/gdebi.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Set an APT configuration option:
+
+`gdebi {{package.deb}} -o {{APT_OPTS}}`
```
Comment (bl-ue) [inline pages/linux/gdebi.md:20]:
```suggestion
`gdebi {{path/to/package.deb}} --option={{APT_OPTS}}`
```

File: pages/linux/gdebi.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Use alternative root dir:
+
+`gdebi {{package.deb}} root=ROOTDIR`
```
Comment (bl-ue) [inline pages/linux/gdebi.md:24]:
```suggestion
`gdebi {{path/to/package.deb}} --root={{path/to/root_dir}}`
```

File: pages/linux/gdebi.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# gdebi
+
+> Simple tool to install deb files.
+> More information: <https://launchpad.net/gdebi>.
+
+- Install local deb packages resolving and installing its dependencies:
+
+`gdebi {{package.deb}}`
```
Comment (bl-ue) [inline pages/linux/gdebi.md:8]:
```suggestion
`gdebi {{path/to/package.deb}}`
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
Hi @prophet1996, thank you for the new page! I've left some comments below for you to review.

1. I think you should use `.deb` instead of deb,
2. We prefer a path rather than just a filename, e.g. `path/to/package.deb` instead of `package.deb`
3. We prefer long options when possible
4. It looks like you forgot the `--` and the token syntax for the `--root` example 🙂

**Review by bl-ue — APPROVED** [review]

File: pages/linux/gdebi.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# gdebi
+
+> Simple tool to install `.deb` files.
+> More information: <https://launchpad.net/gdebi>.
```
Comment (navarroaxel) [inline pages/linux/gdebi.md:4]: I think this is a better online man page:
```suggestion
> More information: <https://www.commandlinux.com/man-page/man1/gdebi.1.html>.
```

**Review by navarroaxel — APPROVED** [review]

**Comment by bl-ue** [comment]
>I think this is a better online man page

@navarroaxel it links to the homepage, too.

**Pull request merged**